src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,51 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Manufacturer ferrari = new Manufacturer(4L, "ferrari", "Italy");
+        Manufacturer volvo = new Manufacturer(5L, "volvo", "USA");
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        manufacturerService.create(ferrari);
+        manufacturerService.create(volvo);
+
+        DriverService driverService = (
+                DriverService) injector.getInstance(DriverService.class);
+        Driver jane = new Driver(4L, "jane","4444");
+        Driver rick = new Driver(6L, "rick", "666666");
+        Driver lina = new Driver(5L, "lina","55555");
```

Comment:
you shouldn't specify an index of Driver to be created. Also let's not use hardcoded ids in your test/example code